### PR TITLE
chore: add adminLayout to admin pages

### PR DESCRIPTION
**Pull request by valerydluski**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [x] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

_Describe the source of requirement, like related issue link._

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/components/AdminSider.tsx [hunk 1]
```diff
@@ -31,125 +32,123 @@ export function AdminSider(props: Props) {
...
-            </a>
-          </Link>
-        </Menu.Item>
-
-        {props.isAdmin ? (
-          <Menu.Item key="2">
-            <Link prefetch={false} href="/admin/courses">
+    <Sider trigger={null} collapsible collapsed={collapsed} theme="light">
+      {collapsed ? <MenuUnfoldOutlined {...menuIconProps} /> : <MenuFoldOutlined {...menuIconProps} />}
+      <Menu mode="inline" style={{ background: '#fff' }}>
```
Comment (AlreadyBored) [inline client/src/components/AdminSider.tsx:37]: What is the purpose of this `style` modification? I don't see visual difference here :)

**Review by AlreadyBored — COMMENTED** [review]
I am not sure that this is due to current changes but there's a buggy title on most of the pages (except Employer page). To reproduce: click on link in the menu of the Sider:
On pages:
* `admin/courses`
* `admin/interview-questions`
* `admin/tasks`
* `admin/events`
* `admin/users`
* `admin/mentor-registry`
* `admin/discord-server`
* `admin/user-group`
Title "Notifications" is present.

**Comment by valerydluski** [comment]
> I am not sure that this is due to current changes but there's a buggy title on most of the pages (except Employer page). To reproduce: click on link in the menu of the Sider: On pages:
> 
> * `admin/courses`
> * `admin/interview-questions`
> * `admin/tasks`
> * `admin/events`
> * `admin/users`
> * `admin/mentor-registry`
> * `admin/discord-server`
> * `admin/user-group`
>   Title "Notifications" is present.

**Comment by valerydluski** [comment]
> I am not sure that this is due to current changes but there's a buggy title on most of the pages (except Employer page). To reproduce: click on link in the menu of the Sider: On pages:
> 
> * `admin/courses`
> * `admin/interview-questions`
> * `admin/tasks`
> * `admin/events`
> * `admin/users`
> * `admin/mentor-registry`
> * `admin/discord-server`
> * `admin/user-group`
>   Title "Notifications" is present.

updated

**Review by AlreadyBored — APPROVED** [review]

**Review by Pulya10c — APPROVED** [review]

**Comment by apalchys** [comment]
@valerydluski have you tried understand what happened?

image

**Comment by valerydluski** [comment]
> @valerydluski have you tried understand what happened?
> 
> image


this is due to the use of SubMenu

**Pull request merged**